code/modules/food/drinkingglass/shaker.dm

```diff
@@ -25,7 +25,7 @@
 
 /obj/item/weapon/reagent_containers/food/drinks/glass2/fitnessflask/proteinshake/Initialize()
 	..()
-	reagents.add_reagent("nutriment", 30)
+	reagents.add_reagent("nutriment", 15)
 	reagents.add_reagent("iron", 10)
-	reagents.add_reagent("protein", 35)
-	reagents.add_reagent("water", 25)
+	reagents.add_reagent("protein", 15)
```

Comment:
Why such a hard nerf to how much protein they contain? it's got value for chemists trying to create more biomass because chemistry is suffering already and xenochimera because they require it to regent after getting fucked over by the timer subsystem deciding that what should have taken three minutes instead gets to take a whopping twenty seven and some change; along with a good aspect of things such as being the cutting edge thing that'll help to stabilize a dying explorer out on the rift

Reply:
Protein shakes are kinda at a powergaming point on the server at this moment. People will rush and steal every single one from multiple vending machines, and this causes crew to run into hunger issues (like for Teshari players). While nerfing it down as much as it is, there was a buff to liquid rations to help counter-act the nerf to the protein shake. After running the numbers through multiple people, people thought my initial drop of 65 units of food to 40 was still too high and most aligned with 30 as the baseline. As this is the first wave, we're buffing a lot of other food options too, and will keep track of how this goes. If there is an immediate issue with hunger or crew we will adjust the protein shake up some to help with balance.

As the numbers for a lot of the food options are.. interesting, this first test it to mainly see how much we ACTUALLY need to adjust if people are struggling with hunger with the first balance.